docker-compose.monitoring.yml

```
@@ -0,0 +1,457 @@
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./monitoring:/monitoring:ro
+    environment:
+      - HEALTH_CHECK_INTERVAL=300  # 5분마다 체크
+      - SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL:-}
+      - EMAIL_ALERTS=${EMAIL_ALERTS:-false}
+    command: |
+      sh -c '
+        apk add --no-cache curl jq
+        echo "Starting automated health monitoring..."
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)